databricks/koalas/indexes.py

```diff
@@ -378,6 +395,30 @@ def name(self) -> str:
     def name(self, name: str) -> None:
         raise PandasNotImplementedError(class_name='pd.MultiIndex', property_name='name')
 
+    def to_pandas(self) -> pd.MultiIndex:
+        """
+        Return a pandas MultiIndex.
+
+        .. note:: This method should only be used if the resulting Pandas object is expected
+                  to be small, as all the data is loaded into the driver's memory. If the input
+                  is large, set max_rows parameter.
+
+        Examples
+        --------
+        >>> df = ks.DataFrame([(.2, .3), (.0, .6), (.6, .0), (.2, .1)],
+        ...                   columns=['dogs', 'cats'],
+        ...                   index=[list('abcd'), list('efgh')])
+        >>> df['dogs'].index.to_pandas()  # doctest: +SKIP
+        MultiIndex([('a', 'e'),
+                    ('b', 'f'),
+                    ('c', 'g'),
+                    ('d', 'h')],
+                   )
+        """
+        return self._kdf[[]]._to_internal_pandas().index
```

Comment:
I maybe missed something but why we cannot do this in `Index` and share the implementation?

Reply:
`Index`'s `scol` might be changed. We need to apply the changes.

Reply:
oh, NVM. got it. We should select `self._scol` in case it has some column expressions there. Don't we have to do the same thing here too?

Reply:
oh, okay, but we don't currently support multi index operations like:

```python
>>> ks.range(10).index + 1
Int64Index([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], dtype='int64')
```

Okay, LGTM but might be good to have some comments here like `TODO:`.

Reply:
So far, we don't have any functions to change the internal state of `MultiIndex` except for series-like operations. In that case, it creates new `Index` object instead of `MultiIndex`.

Reply:
Yes, I checked this assert `assert len(kdf._internal._index_map) == 1`.

At the current moment, the anchor Koalas DataFrame should select Spark column to apply series-like opertations. If we only select from anchor Koalas DataFrame, the Spark column will be lost. Since we don't currently support multi-index, I suspect it's fine for now.